OpenRA.Mods.Common/Effects/Missile.cs

```diff
@@ -22,40 +22,89 @@ namespace OpenRA.Mods.Common.Effects
 {
 	class MissileInfo : IProjectileInfo
 	{
+		[Desc("Name of the image containing the projectile sequence.")]
 		public readonly string Image = null;
+
+		[Desc("Projectile sequence name.")]
 		[SequenceReference("Image")] public readonly string Sequence = "idle";
+
+		[Desc("Palette used to render the projectile sequence.")]
 		public readonly string Palette = "effect";
+
+		[Desc("Should the projectile's shadow be rendered?")]
 		public readonly bool Shadow = false;
+
 		[Desc("Projectile speed in WDist / tick")]
 		public readonly WDist Speed = new WDist(8);
-		[Desc("Maximum vertical pitch when changing altitude.")]
-		public readonly WAngle MaximumPitch = WAngle.FromDegrees(30);
+
 		[Desc("How many ticks before this missile is armed and can explode.")]
 		public readonly int Arm = 0;
+
 		[Desc("Is the missile blocked by actors with BlocksProjectiles: trait.")]
 		public readonly bool Blockable = true;
+
 		[Desc("Maximum offset at the maximum range")]
 		public readonly WDist Inaccuracy = WDist.Zero;
+
 		[Desc("Probability of locking onto and following target.")]
 		public readonly int LockOnProbability = 100;
-		[Desc("In n/256 per tick.")]
-		public readonly int RateOfTurn = 5;
-		[Desc("Explode when following the target longer than this many ticks.")]
+
+		[Desc("Horizontal rate of turn in 360�/256 per tick.")]
```

Comment:
Facing and WAngle units are pretty standard across the code, so i dont think its really appropriate to explain the conversions here.

On a side note, converting the remaining facing units to WAngle is the next ugly giant refactor after height maps, as the current mess is stopping us from doing unit orientations on slopes etc.